ts/Maps/MapViewDefaults.ts

```diff
@@ -0,0 +1,363 @@
+             *
+             * @sample maps/demo/projection-explorer
+             *         Projection explorer
+             * @sample maps/mapview/projection-custom-proj4js
+             *         Custom UTM projection definition
+             * @sample maps/mapview/projection-custom-d3geo
+             *         Custom Robinson projection definition
+             * @sample maps/demo/topojson-projection
+             *         Orthographic projection
+             *
+             * @type   {string}
+             */
+            name: void 0,
+
+            /**
+             * The two standard parallels that define the map layout in conic
+             * projections, like the LambertConformalConic projection. If only
+             * one number is given, the second parallel will be the same as the
+             * first.
+             *
+             * @sample maps/mapview/projection-parallels
+             *         LCC projection with parallels
+             * @sample maps/demo/projection-explorer
+             *         Projection explorer
+             *
+             * @type {Array<number>}
+             */
+            parallels: void 0,
+
+            /**
+             * Rotation of the projection in terms of degrees `[lambda, phi,
+             * gamma]`. When given, a three-axis spherical rotation is be applied
+             * to the globe prior to the projection.
+             *
+             * * `lambda` shifts the longitudes by the given value.
+             * * `phi` shifts the latitudes by the given value. Can be omitted.
+             * * `gamma` applies a _roll_. Can be omitted.
+             *
+             * @sample maps/demo/projection-explorer
+             *         Projection explorer
+             * @sample maps/mapview/projection-america-centric
+             *         America-centric world map
+             */
+            rotation: void 0
+        },
+
+        /**
+         * The zoom level of a map. Higher zoom levels means more zoomed in. An
+         * increase of 1 zooms in to a quarter of the viewed area (half the
+         * width and height). Defaults to fitting to the map bounds.
+         *
+         * In a `WebMercator` projection, a zoom level of 0 represents
+         * the world in a 256x256 pixel square. This is a common concept for WMS
+         * tiling software.
+         *
+         * @sample {highmaps} maps/mapview/center-zoom
+         *         Custom view of a world map
+         * @sample {highmaps} maps/mapview/get-view
+         *         Report the current view of a preprojected map
+         *
+         * @type   {number}
+         */
+        zoom: void 0
+
+    },
+
+    /**
+     * Generic options for the placement and appearance of map insets like
+     * non-contiguous territories.
+     *
+     * @since        10.0.0
+     * @product      highmaps
+     * @optionparent mapView.insetOptions
+     */
+    mapViewInsets: {
```

Comment:
Since this structure is resolved to `mapView.insetOptions` (as seen in the `@optionparent`) tag and [in the API](https://api.highcharts.com/highmaps/mapView.insetOptions), wouldn't it make more sense to place it in that position in the `MapViewDefaults` file too? Then we wouldn't need the `@optionparent` tag, and the top-level `mapView` property?

Reply:
I tried it and it was failing, because the defaults contains the properties for `mapView.insets` and not `mapView.insetOptions`. So the doclets are only partly matching. We could reshuffle the doclets, but I am unsure if this would stop the confusion.